.jshintrc

```diff
@@ -31,7 +31,7 @@
   "undef"         : true,     // Require all non-global variables be declared before they are used.
   "unused"        : true,     // Warn when variables are created but not used.
   "trailing"      : true,     // Prohibit trailing whitespaces.
-  "es3"           : true,     // Prohibit trailing commas for old IE 
+  "es3"           : false,     // Prohibit trailing commas for old IE 
```

Comment:
Should the comment change here as well?

Not crazy about leaking in commas. Harmless I suppose.